build/codegen/codegen.sh

```diff
@@ -14,19 +14,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# shellcheck disable=SC2086,SC2089,SC2090
+# Disables quote checks, which is needed because of the SED variable here.
+
 scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-cd ${scriptdir}/../../vendor/k8s.io/code-generator && ./generate-groups.sh \
+cd "${scriptdir}/../../vendor/k8s.io/code-generator" && ./generate-groups.sh \
   all \
   github.com/rook/rook/pkg/client \
   github.com/rook/rook/pkg/apis \
   "rook.io:v1alpha1,v1alpha2 ceph.rook.io:v1alpha1"
 # this seems busted in the release-1.8 branch
 #  --go-header-file ${SCRIPT_ROOT}/build/codegen/header.txt
 
+SED="sed -i.bak"
```

Comment:
Doesn't this just leave behind a bunch of .bak files? How do they all get cleaned up?

Reply:
Look at line 44. :)
https://github.com/rook/rook/pull/1712/files#diff-d6b28ecdaf1e8f2759c1062ca98f7549R44

Reply:
Ah, I see. Well this works fine on Ubuntu.